grafanalib/core.py

```diff
@@ -398,8 +403,9 @@ def to_json_data(self):
             'editable': self.editable,
             'height': self.height,
             'panels': self.panels,
-            'showTitle': showTitle,
+            'showTitle': self.showTitle,
```

Comment:
I don't understand the motivation for this change. The logic above sets `showTitle` to `True` if someone specifies a title.

Reply:
Let me dig a bit, there was a reason for it. However, `self.showTitle` isn't used otherwise is it (i.e., it's stale code)?

Reply:
The reason I made `showTitle` explicit is two-fold:
1. There is an option `showTitle`, which is otherwise ignored
2. I am porting a dashboard which has a hidden title

Is there any reason not to support point 2?

Reply:
Fully OK with allowing explicit override of `showTitle`. Just want to keep the current behaviour if it's not specified.